src/factories/internal/metadata/iterate_metadata.ts

```diff
@@ -40,6 +40,7 @@ export const iterate_metadata = (props: IMetadataIteratorProps): void => {
     return;
 
   // ITERATE CASES
+  // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
I disabled the lint error for now.
It's up to you if you want to delete it @samchon